src/ListArticles.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import { getApiArticles } from "./getApi";
+import Article from "./Article";
+
+export default function ListArticles() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    getApiArticles().then((response) => {
+      setArticles(response.articles);
+      console.log(response.articles);
+    });
+  }, []);
+  return (
+    <>
+      <ul key={articles}>
+        {articles.map((article) => (
+          <ul key={article.article_id}>
```

Comment:
This `<ul>` on line 17 is inside another `<ul>`

You only need the first one on line 15, the inside on line 17 should be an `<li>`